rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationRector/config/configured_rule.php

```diff
@@ -24,5 +29,6 @@
                 new SimpleStaticType(ReturnOfStatic::class)
             ),
             new AddReturnTypeDeclaration(DataTransformerInterface::class, 'transform', new MixedType()),
+            new AddReturnTypeDeclaration(FormTypeInterface::class, 'getParent', $nullableStringType),
```

Comment:
it need to target to `Source` class location:

```suggestion
            new AddReturnTypeDeclaration(\Rector\Tests\TypeDeclaration\Rector\ClassMethod\AddReturnTypeDeclarationRector\Source\FormTypeInterface::class, 'getParent', $nullableStringType),
```